### PR TITLE
Move check cluster configuration to the begining

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- because we do not have srtages in the upgrade kyma process, the check step must be the first one - to prevend unecessary running other steps after the reconciler is called

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/cmd/broker/main.go [hunk 1]
```diff
@@ -878,6 +878,14 @@ func NewKymaOrchestrationProcessingQueue(ctx context.Context, db storage.BrokerS
 		step     upgrade_kyma.Step
 		cnd      upgrade_kyma.StepCondition
 	}{
+		// check cluster configuration is the first step - to not execute other steps, when cluster configuration was applied
+		// this should be moved to the end when we introduce stages like in the provisioning process
+		// (also return operation, 0, nil at the end of apply_cluster_configuration)
+		{
+			weight: 1,
```
Comment (wozniakjan) [inline components/kyma-environment-broker/cmd/broker/main.go:885]: fwiw, initially it had weight 2 - https://github.com/kyma-project/control-plane/pull/1201/files#r782155263

not sure if it's relevant or if 1 is ok as well. Feel free to unhold if it doesn't matter.

**Review by wozniakjan — APPROVED** [review]
For completion, the bug was introduced in https://github.com/kyma-project/control-plane/pull/1201
Thanks for fixing this!

/lgtm
/hold

**Review by szwedm — APPROVED** [review]

File: components/kyma-environment-broker/cmd/broker/main.go (hunk 1 above)
Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/cmd/broker/main.go:885]: it is not a problem, init step has weight=0

**Pull request merged**